README.md

```diff
@@ -43,6 +55,31 @@ The plugin can also be configured to use multiple password files if required.
 </authConfigs>
 ```
 
+## Generating passwords using htpasswd
+You can use the [htpasswd](http://httpd.apache.org/docs/2.0/programs/htpasswd.html) program from Apache to manage your password file. You must use the -s option with htpasswd to force it to use SHA1 encoding for the password. So for example, you can use the following command to create a password file called "passwd" and put the password for the user "user" in it:
+
+```htpasswd -c -s passwd user```
```

Comment:
I opened this for the release notes. https://github.com/gocd/filebased-authentication-plugin/issues/13